postgresql/run.sh

```diff
@@ -6,8 +6,9 @@ cat queries.sql | while read -r query; do
     sync
     echo 3 | sudo tee /proc/sys/vm/drop_caches
 
-    echo "$query";
-    for i in $(seq 1 $TRIES); do
-        sudo -u postgres psql test -t -c '\timing' -c "$query" | grep 'Time'
-    done;
-done;
+    echo "$query"
```

Comment:
I am fine with the change in this file, however the previous state looked more like Python / C++. I guess we are now using some bash idioms.

Reply:
Yeah, this change is mainly to reuse the same connection for the hot run. I did a similar trick for pg_duckdb.